ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-release-4.10-presubmits.yaml

```diff
@@ -1,5 +1,211 @@
 presubmits:
   openshift-psap/ci-artifacts:
+  - agent: kubernetes
+    always_run: false
```

Comment:
can be turned `on`, so that only `image` runs, and not none of `{nto,sro,nfd,<variant>}-image`